Tests/SentryTests/SentrySpanTests.swift

```diff
@@ -206,4 +215,36 @@ class SentrySpanTests: XCTestCase {
         group.wait()
         XCTAssertEqual(span.data!.count, outerLoop * innerLoop)
     }
+    
+    @available(tvOS 10.0, *)
+    @available(OSX 10.12, *)
+    @available(iOS 10.0, *)
+    func testModifyingTagsFromMultipleThreads() {
```

Comment:
`m`: I think we could merge that test with `testModifyingExtraFromMultipleThreads` and modify both tags and extras in the inner loop.